src/Domain/FunctionsOfTime/PiecewisePolynomial.cpp

```diff
@@ -154,6 +154,30 @@ bool operator!=(const PiecewisePolynomial<MaxDeriv>& lhs,
   return not(lhs == rhs);
 }
 
+template <size_t MaxDeriv>
+std::ostream& operator<<(
+    std::ostream& os,
+    const PiecewisePolynomial<MaxDeriv>& piecewise_polynomial) noexcept {
+  const auto size = piecewise_polynomial.deriv_info_at_update_times_.size();
+
+  auto print_deriv_info =
+      [](std::ostream& local_os,
+         const FunctionOfTimeHelpers::StoredInfo<MaxDeriv + 1>& deriv_info) {
```

Comment:
I suggest you just move this over to `FunctionOfTimeHelpers.hpp`, so it's the stream-operator for `StoredInfo`.

Reply:
I like that suggestion. That way other functions of time can be streamable as well.